### PR TITLE
Composer mode UI updates

### PR DESCRIPTION
**Pull request by stefanceriu**
- Slightly tweak the editing composer mode UI
- Update reply composer mode UI to include message being replied to

Screenshot 2023-05-30 at 10 43 11

https://github.com/vector-im/element-x-ios/assets/637564/c770d11a-3b66-4522-a482-ad84742f0f5a

### CODE REVIEW

**Comment by callumu** [comment]
Everything in the video looks great imo

**Comment by stefanceriu** [comment]
> Can the size of the icons within the message replies be reduced to 16x16?

@callumu Sure, but these views are shared between the composer reply mode and the timeline reply bubbles. Should I change both?

**Comment by stefanceriu** [comment]
> can it be 20x20 instead of what I previously said?

Of course, thanks 👍

**Comment by stefanceriu** [comment]
> One more small thing

Is this what you had in mind? The composers uses 9pts and the timeline 4.

Screenshot 2023-05-30 at 16 20 27

File: ElementX/Sources/Screens/RoomScreen/View/MessageComposer.swift [hunk 1]
```diff
@@ -209,22 +215,47 @@ struct MessageComposer_Previews: PreviewProvider {
...
+                            mode: .edit(originalItemId: UUID().uuidString),
                             sendAction: { },
                             pasteAction: { _ in },
                             replyCancellationAction: { },
                             editCancellationAction: { })
         }
         .padding(.horizontal)
+        
+        ScrollView {
+            VStack {
```
Comment (Velin92) [inline ElementX/Sources/Screens/RoomScreen/View/MessageComposer.swift:227]: since this contains a for each loop wouldn't be better to use a LazyVStack ?

**Review by Velin92 — APPROVED** [review]

File: ElementX/Sources/Screens/RoomScreen/View/MessageComposer.swift (hunk 1 above)
Comment (stefanceriu) [reply ElementX/Sources/Screens/RoomScreen/View/MessageComposer.swift:227]: I guess, but it's just a preview so who cares? 😁

Comment (Velin92) [reply ElementX/Sources/Screens/RoomScreen/View/MessageComposer.swift:227]: oh yeah mb, that's preview so it's statically defined.

**Pull request merged**